src/app/config.ts

```diff
@@ -104,6 +124,7 @@ export class AltairConfig {
     this.initialData.query = (window as any).__ALTAIR_INITIAL_QUERY__ || initialQuery || '';
     this.initialData.variables = (window as any).__ALTAIR_INITIAL_VARIABLES__ || initialVariables || '';
     this.initialData.headers = (window as any).__ALTAIR_INITIAL_HEADERS__ || initialHeaders || '';
+    this.initialData.environments = (initialEnvironments || {}) as IInitialEnvironments;
```

Comment:
Did we need to specify the `as IInitialEnvironments` here?

Reply:
I was getting a TypeScript error otherwise because the fallback empty object doesn't actually implement the `IInitialEnvironments` interface. Is there a better way to handle this?

Reply:
I would say set it as `null` but I'm not sure how Typescript would react to that either.